src/worker.js

```diff
@@ -495,33 +567,62 @@
       var self = this;
       var expirationDate = new Date(self.initData.authTokenExpiration);
       var currentTimeStamp = new Date().getTime();
-      var fiveMins = 5 * 60 * 1000;
+      var thirtyMins = 30 * 60 * 1000;
```

Comment:
constants should be capitalized, and preferably contain the unit of the value, THIRTY_MINUTES_MS

Reply:
Agreed. Its just to keep the old standards in sync  -  if its not a must have i would like to take this as part of next commit.